### DISCUSSION TITLE
utilizing the package

### DISCUSSION THREAD

**Discussion by AliSayyah** [AliSayyah/django-urlconfchecks#23] [Ideas]
It would be helpful to have a better solution than simply importing the package.
I would appreciate your thoughts.

**Comment by spookylukey** [comment]
The mechanism you've chosen seems to work well.

The only other mechanism I can think of is to put something into `INSTALLED_APPS`. That might be slightly better, if you can get it to work, because I think people will expect that more.

**Reply by AliSayyah** [reply]
Yes, that would be better. I need to look into how Django manages apps and leverage that.

**Reply by spookylukey** [reply]
I think it might do: `import <yourapp>.apps` if that exists, so you may be able to use that, otherwise `<yourapp>.models`

**Reply by AliSayyah** [reply]
With your help, now package can be used as a Django app.
Thank you.
